app/assets/stylesheets/utils/_mixins.scss

```diff
@@ -1,28 +1,28 @@
-/** For easier size breakpoints. */
-
-$breakpoints: (
-  'tiny':   ( max-width:  360px ),
-  'small':  ( max-width:  500px ),
-  'mobile': ( max-width:  745px ),
-  'medium': ( max-width:  990px ),
-  'large':  ( max-width: 1040px )
-);
-
-@mixin breakpoint($name) {
-  @if map-has-key($breakpoints, $name) {
-    @media #{inspect(map-get($breakpoints, $name))} {
-      @content;
-    }
-  }
-  @else {
-    @media ("#{$name}") {
-      @content;
-    }
-  }
-}
-
-@mixin hidpi {
-  @media (-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi) {
-    @content;
-  }
-}
+// Easier media query breakpoints
+$breakpoints: (
+  'tiny':   ( max-width:  360px ),
+  'small':  ( max-width:  500px ),
```

Comment:
Expected 0 spaces after `(` instead of ` `<br>Expected 0 spaces before `)` instead of ` `